bindings/qt/lib/src/toolkit.cpp

```diff
@@ -37,15 +37,14 @@ int Toolkit::adjustedDisplayHeightForPage(int page)
     }
     else {
         int averagePageHeight = 0;
-        int defaultPageHeight = 1000;
```

Comment:
I find it clearer to have it set separately and not within an instruction. But you can add `const` if you want.